website/docs/docs/dbt-versions/core-upgrade/01-upgrading-to-v1.8.md

```diff
@@ -28,7 +28,14 @@ dbt Labs is committed to providing backward compatibility for all versions 1.x,
 
 ## New and changed features and functionality
 
-Coming soon!
+### Additional attributes in run_results.json
+
+There are three attributes related to the `applied` state in the run_results.json (to complement unique_id):
```

Comment:
Should this be a partial / snippet because it's in two places

Reply:
I don't think we need to  in this scenario because it's being reused only in the upgrade guide, which will not be changed if these features expand or change in the future; they'd be put in a new upgrade guide for that version.